packages/design-system-react/src/components/icon/icons/AddSquare.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
In a previous version of this PR I added the `icons/` folder to the .gitignore because they are all automatically generated but we need them for tests